src/ucp/core/ucp_context.h

```diff
@@ -126,20 +126,34 @@ typedef struct ucp_tl_alias {
 } ucp_tl_alias_t;
 
 
+/**
```

Comment:
Maybe component can contain MDs? Or will it be too many indexing changes thru the code?

Reply:
i want to have global indexing for MDs to avoid the need to keep both component index and md index